output/typescript/types.ts

```diff
@@ -7855,6 +7866,14 @@ export interface OpenJobResponse extends ResponseBase {
   opened: boolean
 }
 
+export interface OpenPointInTimeRequest extends RequestBase {
+  index: IndexName
+  keep_alive?: Time
+}
+
+export interface OpenPointInTimeResponse extends ResponseBase {
+}
```

Comment:
Should this not include the returned ID?

Reply:
done with https://github.com/elastic/elastic-client-generator/pull/215/commits/c0a2c666de5c642c73b33602abf3d896bc598a65 - great catch, thanks!

Reply:
Shouldn't this have been caught via validator? Or are there no tests for PIT APIs?

Reply:
I had the same train of thought, but thank god we have @stevejgordon :)

Reply:
Haha! Thank God I'd recently been poking around those APIs! 😄